app/views/course/assessment/submission/submissions/_statistics.html.slim

```diff
@@ -11,11 +11,20 @@ div.panel.panel-default
           td = @submission.workflow_state
         tr
           th = @submission.class.human_attribute_name(:grade)
-          td = @submission.grade
+          td
+            span#submission-statistics-total-grade>
+              = @submission.grade
+            ' /
+            span#submission-statistics-maximum-grade
+              = @submission.assessment.maximum_grade
         tr
           th = @submission.experience_points_record.class.human_attribute_name(:points_awarded)
           - if can?(:grade, @submission)
-            td = f.input :points_awarded, label: false
+            / TODO: Factor in time-based experience points
+            td
+              = f.input :points_awarded, label: false,
+                        input_html: { class: 'submission-points-awarded',
+                                      'data-base-points' => @submission.assessment.base_exp }
```

Comment:
If already graded, the value that will be displayed is `points_awarded` not `base_exp`. `data-base-points` is the set here so that we can calculate `points_awarded` when the grades change.